
show_config: true
debug: true

ws_x_max: 27
ws_y_max: 4
ws_z_max: 5

ws_x_min: -28
ws_y_min: -1
ws_z_min: 0
map_resolution: 0.2
map_h_inflaction: 0.6
map_v_inflaction: 0.2
z_weight_cost: 1
z_not_inflate: 3
goal_weight: 1.3
traj_dxy_max: 1
traj_pos_tol: 1
traj_yaw_tol: 0.1
traj_dz_max: 1
traj_vxy_m: 1
traj_vz_m: 1
traj_vxy_m_1: 1
traj_vz_m_1: 1
traj_wyaw_m: 1
world_frame: map
robot_base_frame: base_link